src/shared/Core/Interop/Posix/PosixEnvironment.cs

```diff
@@ -34,7 +34,7 @@ protected override string[] SplitPathVariable(string value)
 
         #endregion
 
-        private static IReadOnlyDictionary<string, string> GetCurrentVariables()
+        protected static IReadOnlyDictionary<string, string> GetCurrentVariables()
```

Comment:
We don't need to make this `protected` if we don't need now to call `GetCurrentVariables()` in derived types 😉